tests/functional/simple_reference/test_invalid_reference.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from tests.util import run_dbt
+from dbt.exceptions import CompilationException
+
+
+descendant_sql = """
+-- should be ref('model')
+select * from {{ ref(model) }}
+"""
+
+
+model_sql = """
+select 1 as id
+"""
+
+
+@pytest.fixture
+def models():
+    return {
+        'descendant.sql': descendant_sql,
+        'model.sql': model_sql,
+    }
+
+
+def test_undefined_value(project):
+    # Tests that a project with an invalid reference fails
+    with pytest.raises(CompilationException):
+        run_dbt(['compile'])
```

Comment:
- This test is much more modular than the others, and neatly maps to a functionality owned by the Language team. Is there some way we could tag this test with that team's ownership? Point back to the actual functionality it's testing?
- Related to the above: Would we want this to be `'parse'` or `'ls'`, rather than `'compile'`, since we'd expect this error to raise during parse time?

Reply:
Those sorts of changes sounds like an excellent idea. Not sure whether we want to do it when converting though, or as a post-conversion re-organization and cleanup.